src/components/SidebarNavItem/SidebarNavItem.js

```diff
@@ -36,14 +42,15 @@ class SidebarNavItem extends PureComponent {
       children,
       pathname, // path from window.location object
       pathName, // name from metadata to use as path for link
-      theme
+      theme,
+      match = { url: '/' } // comes from React Router
     } = this.props;
     const defaultPath = pathName ? pathName : name;
     const activeCss = this.onGetActive(defaultPath, pathname);
     const path = pathName ? pathName : name;
     return (
       <RouterLink
-        to={path}
+        to={`${match.url}${path}`}
```

Comment:
I worry about ending up with 0 or 2 `/` in some cases here

Reply:
Yeah... maybe this component shouldn't actually care about `match` and that should be handled in the app